### ISSUE TITLE
While forwarding a message, the two buttons before the chat list are not labeled

### ISSUE THREAD

**Issue by FrayxRulez** [UnigramDev/Unigram#1445]
https://t.me/unigraminsiders/270748

**Comment by MarcAnt01** [comment]
Missing tooltips?

**Comment by FrayxRulez** [comment]
Fixed in https://github.com/UnigramDev/Unigram/commit/dbb7035bb95f16a0a96381e2b4a533e4479b65a4
